source/Halibut.Tests/FailureModesFixture.cs

```diff
@@ -138,6 +139,8 @@ public async Task FailWhenServerThrowsDuringADataStream(ServiceConnectionType se
                        .ForServiceConnectionType(serviceConnectionType)
                        .WithReadDataStreamService()
                        .WithPollingReconnectRetryPolicy(() => new RetryPolicy(1, TimeSpan.Zero, TimeSpan.Zero))
+                       // This test is very noisy for the test logs
+                       .WithHalibutLoggingLevel(LogLevel.Fatal)
```

Comment:
Some of the tests are generating 1000s of Halibut log messages and this was killing the WIndows Test Build on Teamcity.